docker-compose.yaml

```diff
@@ -145,7 +145,7 @@ services:
 
   keycloak-config:
     container_name: keycloak_config
-    image: bitnami/keycloak-config-cli:6.1.6
```

Comment:
Similar to the comment above, we're going to plan this update later along with some thorough testing.